index.html

```diff
@@ -1,214 +1,212 @@
+                        <div style="padding: 10px 10px;display: flex;flex-direction: column;">
+                            <ul class="text-capitalize what-we-do" style="margin: auto;margin-top: 20px;list-style-image: url(&quot;/assets/img/bullet.png&quot;);width: 100%;">
+                                <li>Click “Backup now” to save a backup as a .tabfern file in our Downloads directory.</li>
+                                <li>Click “Load contents of a backup”, then choose the file ou want and hit OK, to add the backed-up windows to your tree.</li>
+                                <li>Loading a backup is not a “restore” operation that takes ou back to where you were. Instead, it <em>adds</em> the loaded information to the windows and tabs you already have open or saved. That way you don’t have to worry
+                                    about losing your current place.</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <div style="display: flex;"><img style="width: 60%;height: 100%;margin: auto;margin-top: 2vh;" src="/assets/img/hamburger-screenshot.png"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div style="margin-bottom: 50px;">
+        <div class="container" style="padding: 0px;max-width: 1400px;">
+            <div>
+                <div class="row rower" id="limit">
+                    <div class="col">
+                        <div id="usage">
+                            <header>
+                                <h1 style="font-size: 64px;">Limits</h1>
+                            </header>
+                            <ul class="usage-list" style="margin: 1vw;margin-top: 20px;list-style-image: url(&quot;/assets/img/bullet.png&quot;);/*width: 80%;*/">
+                                <li>
+                                    <div>
+                                        <p style="margin: 0px;">There is only a two-level hierarchy — tabs cannot be the children&nbsp;of other tabs in the tree.</p>
+                                    </div>
+                                </li>
+                                <li>You cannot open and close individual tabs — you have to open and close the window as a whole.</li>
+                                <li>Windows will not necessarily open at the size of the last-closed window,&nbsp;like they do normally in Chrome.</li>
+                            </ul>
+                            <p style="font-size: 18px;">See the&nbsp;&nbsp;<a href="https://github.com/cxw42/TabFern/issues">issue tracker</a>&nbsp; on GitHub for&nbsp;more about current limitations, and where TabFern is going in the future!</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div style="background-color: #7E544D;color: white;">
```

Comment:
Why is this div brown instead of blue?  Could we make these regular sections with a white background, just like "Backup & Restore" and the others?

Reply:
I wanted to showcase the footer in a different color. Hence showing the site in a three-color scheme - Blue, White, and Brown. If you suggest blue for it too, then I'll make it blue

Reply:
I would personally prefer blue/white/blue or green/white/green - I don't think the footer information needs to stand out particularly.  Thank you!